.gitignore

```diff
@@ -1,2 +1,2 @@
-*/.idea/
+*.idea/
```

Comment:
Just curious why this change

Reply:
Because a `git status` shows this

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	.idea/
```

I think when I initially made the change to add this I had only opened up the `monitor-translations` folder so it created an .idea in there and that led to `*/.idea/` working.  Now that I've opened up the main `salus-data-loader-content` folder the .idea at the top level isn't caught by the original ignore.